feature/qos/otg_tests/ingress_traffic_classification_and_rewrite_test/README.md

```diff
@@ -0,0 +1,91 @@
+# DP-1.16: Ingress traffic classification and rewrite
+
+## Summary
+
+This test aims to validate the functionality of ingress traffic classification and subsequent packet remarking (rewrite) on a Device Under Test (DUT). The DUT's configuration will be evaluated against the OpenConfig QOS model, and traffic flows will be analyzed to ensure proper classification, marking, and forwarding.
+
+## Topology
+
+* ATE:port1 <-> port1:DUT:port2 <-> ATE:port2
+
+## Procedure
+
+* DUT Configuration:
```

Comment:
Please add headings in the style of DP-1.16.x for each section as per the [template](https://github.com/openconfig/featureprofiles/blob/main/doc/test-requirements-template.md#procedure).

Each section should read like a 'subtest' (mapping to function in the code with a pass/fail result).  We are only testing the features of interest on the DUT.  The pre-requisite configuration of the test environment (ATE and DUT) should be in the `Test Environment Setup` section.   So you should split out the things like config of interfaces and routing protocols into the `Test Environment Setup` and keep separate from the gnmi.Set for QoS config push and the traffic verifications which can be organized as you have below into DP-1.16.x subtests.